tests/integration_tests/http_api_tls_with_user_auth/util/test_case.py

```
@@ -208,15 +208,29 @@ def update_changefeed():
     url = BASE_URL0 + "/changefeeds/changefeed-test1?keyspace=keyspace1"
-    data = json.dumps({"mounter_worker_num": 32})
+    data = json.dumps({
+        "replica_config": {
+            "ignore_ineligible_table": True,
+            "mounter": {
+                "worker_num": 32
+            }
+        }
+    })
     headers = {"Content-Type": "application/json"}
     resp = rq.put(url, data=data, auth=Auth,
```

Ruff (0.14.14)

Probable use of `requests` call without timeout

(S113)